### PR TITLE
Support infrabox.yaml

### PR DESCRIPTION
**Pull request by iberryful**

### CODE REVIEW

**Comment by ib-steffen** [comment]
Please also add a e2e test to infrabox/test/e2e, after we have merged the change we also need an example in https://github.com/sap/infrabox-examples and have to update the CLI to also support it.

**Comment by iberryful** [comment]
e2e test is and example are added. CLI is updated too. [https://github.com/SAP/InfraBox-cli/pull/10](PR)

File: infrabox/test/e2e/test.py [hunk 1]
```diff
@@ -263,6 +263,8 @@ def test_insecure_env(self):
         self.run_it('/infrabox/context/infrabox/test/e2e/tests/docker_insecure_env')
         self.expect_job('test')
 
+    def test_infrabox_yaml(self):
+
```
Comment (ib-steffen) [inline infrabox/test/e2e/test.py]: something missing here?

Comment (iberryful) [reply infrabox/test/e2e/test.py]: resolved

**Comment by ib-steffen** [comment]
tests are failing

**Pull request merged**